hr_recruitment_security/tests/test_access_rights.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import logging
+from odoo import fields
+from odoo.exceptions import AccessError
+from odoo.tests.common import SavepointCase
+
+_logger = logging.getLogger(__name__)
+
+
+class TestHrRecruitmentSecurity(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestHrRecruitmentSecurity, cls).setUpClass()
+        # Users
+        Users = cls.env["res.users"].with_context(no_reset_password=True)
```

Comment:
In Python, we should use lowercase for variables. Maybe use `users` or something similar you find appropriate. https://www.python.org/dev/peps/pep-0008/#function-and-variable-names